pages/linux/speedread.md

```diff
@@ -0,0 +1,29 @@
+# speedread
+
+> A simple terminal-based open source Spritz-alike.
+> Shows input text as a per-word RSVP (rapid serial visual presentation) aligned on optimal reading points, which allows reading text at a much more rapid pace than usual as the eye can stay fixed on a single place.
+> More information: <https://github.com/pasky/speedread>.
+
+- Basic example.  Read `tea.txt` at a speed of 250 wpm:
+
+`cat tea.txt | speedread -wpm {{250}}`
```

Comment:
```suggestion
- Read a text file at a speed of 250 wpm:

`cat {{path/to/file.txt}} | speedread -wpm {{250}}`
```

A few suggestions:

- We don't follow the "Basic example" convention across our pages so we can remove it.
- We can generalize the description and also update the input to a path placeholder to align with our other pages.